{"name": "Создание нового сотрудника", "status": "broken", "statusDetails": {"message": "NameError: name 'attempt' is not defined", "trace": "dummy_page = <pages.dummy_page.DummyPage object at 0x7ac12deb20f0>\n\n    @allure.story('Тестирование api')\n    @allure.title('Создание нового сотрудника')\n    @allure.description('По ендпоинту создать сотрудника https://dummy.restapiexample.com/api/v1/create')\n    def test_create_employess(dummy_page):\n        with allure.step('Создание нового сотрудника и проверка то что он создан'):\n>           dummy_page.createEmployee()\n\ntest_api.py:17: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <pages.dummy_page.DummyPage object at 0x7ac12deb20f0>\n\n    def createEmployee(self):\n        response=self.page.request.post(settings.POST_CREATE_EMPLOYEE_DUMMY_URL, data = json.dumps(model.dataCreate), headers=model.headers)\n        logging.info(f'Статус код: {response.status}')\n        if response.status == 429:\n             time.sleep(5)\n             response=self.page.request.post(settings.POST_CREATE_EMPLOYEE_DUMMY_URL, data = json.dumps(model.dataCreate), headers=model.headers)\n    \n        content_type = response.headers.get('content-type', '')\n        logging.info(f'Content-Type:\" {content_type}')\n        if 'application/json' in content_type:\n            try:\n                data = response.json()\n                logging.info(f'JSON Response: {data}')\n                post_id = data.get(\"data\", {}).get(\"id\")\n                if post_id:\n                    logging.info(f'Созданный ID сотрудника: {post_id}')\n                    get_url = settings.GET_EMPLOYEE_DUMMY_URL.replace('&',str(post_id))\n                    print(get_url)\n                    get_response = self.page.request.get(get_url)\n                    if get_response.status == 429:\n>                       logging.warning(f\"Попытка {attempt + 1}: Получен 429. Жду 5 секунд...\")\nE                       NameError: name 'attempt' is not defined\n\n../pages/dummy_page.py:43: NameError"}, "description": "По ендпоинту создать сотрудника https://dummy.restapiexample.com/api/v1/create", "steps": [{"name": "Создание нового сотрудника и проверка то что он создан", "status": "broken", "statusDetails": {"message": "NameError: name 'attempt' is not defined\n", "trace": "  File \"/home/hama/Документы/PythonTests/playwright_project_2/tests/test_api.py\", line 17, in test_create_employess\n    dummy_page.createEmployee()\n  File \"/home/hama/Документы/PythonTests/playwright_project_2/pages/dummy_page.py\", line 43, in createEmployee\n    logging.warning(f\"Попытка {attempt + 1}: Получен 429. Жду 5 секунд...\")\n                               ^^^^^^^\n"}, "start": 1748711808773, "stop": 1748711811755}], "attachments": [{"name": "log", "source": "ada520aa-d59c-4609-b00f-682d2a6bbf74-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "36a566fa-aede-45d2-b431-d664d2e08db7-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "5cfbc266-5fc4-4baa-8a39-e9f7ec902780-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "browser_name", "value": "'firefox'"}], "start": 1748711808772, "stop": 1748711811759, "uuid": "b85edbc2-d467-4b73-8da0-17845ecc6c3b", "historyId": "25b2cad223fabcceac01be3b076e21b5", "testCaseId": "956d3f72c027d6f6df39f72649adb257", "fullName": "tests.test_api#test_create_employess", "labels": [{"name": "story", "value": "Тестирование api"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_api"}, {"name": "host", "value": "hama-ZenBook-UX425IA-UM425IA"}, {"name": "thread", "value": "37507-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_api"}]}